### ISSUE TITLE
Use touch-screen on Win98 Toshiba laptop?

### ISSUE THREAD

**Issue by BenTrem** [dcs-bios/dcs-bios#107]
Apologies if this is way too far off topic.

I have this lovely ToughBook sitting on a shelf. I did yoeman duty for me year after year.
I'd love to use its functionality.
**I would _really hate_ to just junk it!**

I'm not assuming that this would be simple or easy, but success would be fabulous! Even having a single Thrustmaster MFD on it ... I'm motivated!
I imagine it connecting by LAN.

If someone is certain that this is a **_NoGo,_** that would be helpful.
If someone's pretty sure that this might work, that too would help.

thanks in advance
--ben _aka_ **Dawks _[PIR]_**

**Comment by jboecker** [comment]
DCS-BIOS currently does not implement a glass cockpit. I'd like to support something like this at some point in DCS-BIOS 2.x, but that's just an idea on paper at this time.

Right now, you can look at programs like Helios and Ikarus. As far as I know, both of these support displaying the panel on a different computer.

Getting the MFCD contents to display will probably be a bit trickier. You'd have to edit some Lua files to make DCS: World display the MFCD on another monitor *on the same PC that is running DCS* (it doesn't have to be a real monitor, it could be a virtual one or a real port on your GPU with no monitor attached), and then find some way (VNC, Remote Desktop or similar) to stream (part of) that screen to your laptop.

In other words, DCS-BIOS is not the right tool for this job right now, but it's definitely not impossible to make this work.
